.gitignore

```diff
@@ -12,3 +12,4 @@
 ###> Other files and folders ###
 .idea
 ###< Other files and folders ###
+*.lock
```

Comment:
This file should be kept as it prevents versioning issues for other developers cloning the repository, so if you clone it and then run `composer install` you should have the same versions I'm using in the YouTube tutorial

Reply:
Alright, I've correct this part to avoid possible errors.